ibraheemosule/skoolizy
# Za - remember-login-details

# Why this PR?
This PR implements the "Remember Me" functionality for the login form with security. Login details ( password) are encrypted before being saved to localStorage. When a user returns to the login page, the encrypted password is decrypted and used to populate the login fields.

# What does this PR do?
This PR implements the "Remember Me" functionality for the login form with encrypted data storage. 

# How to test your work?
-Verify that login details are encrypted and saved correctly when "Remember Me" is selected.
-Confirm that login fields are pre-filled with decrypted data on revisiting the page.


# Pages to see your changes?
/auth/login
